common/service/config/config.go

```diff
@@ -210,7 +210,7 @@ type (
 		ConnectProtocol string `yaml:"connectProtocol" validate:"nonzero"`
 		// ConnectAttributes is a set of key-value attributes to be sent as part of connect data_source_name url
 		ConnectAttributes map[string]string `yaml:"connectAttributes"`
-		// MaxQPS the max request rate on this datastore
+		// MaxQPS is the max request rate to this datastore
```

Comment:
I think you need to rebase - current master this is static config. 

If you are up for fixing the bug with double creation of resource bean on startup we can include the diff to make this dynamic config in this diff.